stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MoshiApi.kt

```diff
@@ -0,0 +1,8 @@
+package io.getstream.chat.android.client.api2
+
+/**
+ * Temporary marker interface for Moshi based Retrofit API classes.
+ */
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.CLASS)
+internal annotation class MoshiApi
```

Comment:
Used to identify Retrofit interfaces that are used with Moshi - needed so that the Moshi parser is used with these instead of the Gson parser